src/mame/drivers/namcos2.cpp

```diff
@@ -2703,11 +2703,14 @@ ROM_START( finallap )
 	/* no external MCU ROM? previously loaded type C, but the game predates it */
 
 	ROM_REGION( 0x400000, "sprite", 0 ) /* Sprites */
-	ROM_FILL( 0, 0x200000, 0xff )
-	ROM_LOAD32_BYTE( "obj-0b",  0x200003, 0x80000, CRC(c6986523) SHA1(1a4b0e95ade6314850b6e44f2debda0ab6e91397) )
-	ROM_LOAD32_BYTE( "obj-1b",  0x200002, 0x80000, CRC(6af7d284) SHA1(c74f975c301ff15040be1b38359624ec9c83ac76) )
-	ROM_LOAD32_BYTE( "obj-2b",  0x200001, 0x80000, CRC(de45ca8d) SHA1(f476ff1719f60d721d55fd1e40e465f48e7ed019) )
-	ROM_LOAD32_BYTE( "obj-3b",  0x200000, 0x80000, CRC(dba830a2) SHA1(5bd899b39458978dd419bf01082782a02b2d9c20) )
+	ROM_LOAD32_BYTE( "obj-0b",  0x000003, 0x80000, CRC(c6986523) SHA1(1a4b0e95ade6314850b6e44f2debda0ab6e91397) )
+	ROM_RELOAD(0x200003, 0x80000)
```

Comment:
I believe you can just reload the full range with `ROM_COPY( "sprite", 0x000000, 0x200000, 0x200000 )` instead of repeating `ROM_RELOAD` four times.